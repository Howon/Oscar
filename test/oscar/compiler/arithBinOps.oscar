def main() => unit = {
  println(1 + 5);
  println(2 - 5);
  println(3 * 5);
  println(4 / 5);
  println(5 % 5);
}
