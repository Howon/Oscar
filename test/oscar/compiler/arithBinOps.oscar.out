6
3.0000000000
-3
15
0
0.3333333333
0
